src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -1,29 +1,29 @@
 package org.openmrs.contrib.qaframework.automation;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.ActiveVisitsPage;
+import org.openmrs.contrib.qaframework.page.AddOrEditAllergyPage;
+import org.openmrs.contrib.qaframework.page.AllergyPage;
+import org.openmrs.contrib.qaframework.page.AttachmentsPage;
+import org.openmrs.contrib.qaframework.page.ConditionPage;
+import org.openmrs.contrib.qaframework.page.ConditionsPage;
+import org.openmrs.contrib.qaframework.page.PatientVisitsDashboardPage;
+import org.openmrs.contrib.qaframework.page.RequestAppointmentPage;
+import org.openmrs.contrib.qaframework.page.VisitNotePage;
+
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 
-import org.openmrs.contrib.qaframework.RunTest;
-import org.openmrs.reference.page.ActiveVisitsPage;
-import org.openmrs.reference.page.AddOrEditAllergyPage;
-import org.openmrs.reference.page.AllergyPage;
-import org.openmrs.reference.page.AttachmentsPage;
-import org.openmrs.reference.page.ConditionPage;
-import org.openmrs.reference.page.ConditionsPage;
-import org.openmrs.reference.page.PatientVisitsDashboardPage;
-import org.openmrs.reference.page.RequestAppointmentPage;
-import org.openmrs.reference.page.VisitNotePage;
-import org.openmrs.uitestframework.test.TestData;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
```

Comment:
Likes i said above , we cant have an intro ticket to address  these re ordering  of imports,before having this merged.In otherwords this should be the parent ticket where anything should be built from.

Reply:
> is re-ordering imports a priority in this PR?

Not a priority in this Pr Lets fix them in a new pull request via a new ticket, Btw i don't know whether you understand my point.

Reply:
> Not a priority in this Pr 

@sherrif10 re-ordering imports like in Steps file and other files is adding more workload for the reviewer.  And a reviewer has to go through all the files, if we can ignore re-ordering of imports it will reduce files to be reviewed.

Ian mentioned it [here](https://github.com/openmrs/openmrs-contrib-qaframework/pull/136#discussion_r716676776) sometime back and your [response](https://github.com/openmrs/openmrs-contrib-qaframework/pull/136#discussion_r716688001) makes any reviewer know you got the point home.

Reply:
@sherrif10 why aren't you humble?

Reply:
Appologies my Bad , I had mistakenly all long mis-contemplating about the imports suggestion till i re-read the comments.